Common/src/main/java/com/agilepro/commons/UserRole.java

```diff
@@ -33,6 +33,27 @@
 	PROJECT_GROUP(true),
 
 	/**
+	 * Represents Members group under which project members related roles should
+	 * be defined.
+	 **/
+	MEMBER_GROUP(true),
+	
+	/** 
+	 * The admin. 
+	 **/
+	ADMIN,

```

Comment:
Where are we using these roles - ADMIN, MEMBER and Manager

Rename MEMBER_GROUP to PROJ_MEMBER_GROUP and move it down where proj member roles are defined.